proto/messages.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package scheduler;
+
+// Simple WorkLoad description
+message WorkLoad {
```

Comment:
I think the workload message should also contain a serialized JSON for the description of the workload. Then it's up to the node worker to deserialize when receiving the message. So this should look like:

```proto
message Workload {
     string instance_id = 1; // Instance UUID
     string definition = 2; // Workload definition as a serialized JSON
}
```

Reply:
You are right, doing this we won't have to define again the proto file when a new property is needed in a workload definition! :D

@croumegous @MaloPolese There is a little update of the proto for communication between controller & scheduler. See above.